API/controllers/authControllers.go

```diff
@@ -77,6 +103,89 @@ var CreateAccount = func(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// swagger:operation POST /api/users/bulk auth CreateBulk
+// Create multiples users with one request.
+// ---
+// produces:
+// - application/json
+// parameters:
+//   - name: name
+//     in: json
+//     description: User name
+//     type: string
+//     required: false
+//     default: "John Doe"
+//   - name: email
+//     in: body
+//     description: User Email Address
+//     type: string
+//     required: true
+//     default: "user@email.com"
+//   - name: password
+//     in: json
+//     description: User password
+//     required: true
+//     format: password
+//     default: "secret123"
+//
+// responses:
+//
+//	'200':
+//	    description: Request processed, check response body for results
+//	'400':
+//	    description: Bad request
+//	'500':
+//	    description: Internal server error
+var CreateBulkAccount = func(w http.ResponseWriter, r *http.Request) {
+	fmt.Println("******************************************************")
+	fmt.Println("FUNCTION CALL: 	 CreateBulkAccount ")
+	fmt.Println("******************************************************")
+	DispRequestMetaData(r)
+
+	var accounts []models.Account
+	err := json.NewDecoder(r.Body).Decode(&accounts)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		u.Respond(w, u.Message(false, "Invalid request"))
+		return
+	}
+
+	callerUser := getUserFromToken(w, r)
+	if callerUser == nil {
+		return
```

Comment:
shouldnt we respond in that case ?

Reply:
Yes and we do! getUserFromToken takes care of that for us, to avoid copying and pasting the same lines everywhere to respond the same thing in case of error.